src/server/auth/cmds/cmds.go

```diff
@@ -79,8 +94,141 @@ func LoginCmd() *cobra.Command {
 	return login
 }
 
+// CheckCmd returns a cobra command that sends an "Authorize" RPC to Pachd, to
+// determine whether the specified user has access to the specified repo.
```

Comment:
My plan was that only the owner of a repo could get the ACL for that repo, but any user could check if they had access to it